docs/callbacks.rst

```diff
@@ -27,10 +27,10 @@ Simply define a callback function:
     from django.conf import settings
     from django_hosts import patterns, host
 
-    host_patterns = patterns('',
-        host(r'www', settings.ROOT_URLCONF, name='www'),
-        host(r'(?P<username>\w+)', 'path.to.custom_urls',
-             callback='path.to.custom_fn', name='with-callback'),
+    host_patterns = patterns(
+        "",
+        host(r"www", "urls.default", name="default"),
+        host(r"(?P<username>\w+)", "path.to.custom_urls", callback="path.to.custom_fn", name="with-callback"),
```

Comment:
What's the reason for this change (i.e. replacing `settings.ROOT_URLCONF` with `"urls.default"`, and `www` with `default`)? Should we change it back to the original version, or is the new version preferable in some way? 🙂